backend/s3mem/backend.go

```diff
@@ -298,6 +299,43 @@ func (db *Backend) DeleteMulti(bucketName string, objects ...string) (result gof
 	return result, nil
 }
 
+func (db *Backend) DeleteMultiVersions(bucketName string, objects ...gofakes3.ObjectID) (result gofakes3.MultiDeleteResult, err error) {
+	db.lock.Lock()
+	defer db.lock.Unlock()
+
+	bucket := db.buckets[bucketName]
+	if bucket == nil {
+		return result, gofakes3.BucketNotFound(bucketName)
+	}
+
+	now := db.timeSource.Now()
+
+	for _, object := range objects {
+		var dresult gofakes3.ObjectDeleteResult
+		var err error
+		if object.VersionID != "" {
+			_, err = bucket.rmVersion(object.Key, gofakes3.VersionID(object.VersionID), now)
+		} else {
+			dresult, err = bucket.rm(object.Key, now)
+			_ = dresult // FIXME: what to do with rm result in multi delete?
+		}
+
+		if err != nil {
+			errres := gofakes3.ErrorResultFromError(err)
+			if errres.Code == gofakes3.ErrInternal {
+				// FIXME: log
```

Comment:
What are we going to to with these FIXME's?

Reply:
I've adapted this method from [DeleteMulti](https://github.com/Kucukaslan/gofakes3/blob/373b40cfa273f0598085348407a1014280b20a60/backend/s3mem/backend.go#L269-L300) method. I preserved FIXMEs because they seemed still relevant to this method and a decision/change made to DeleteMulti should also be applied here too.

Reply:
👍